templates/layouts/layout.html.twig

```diff
@@ -26,6 +26,10 @@
             <meta name="robots" content="noindex">
         {% endif %}
 
+        {% if page.canonical %}
```

Comment:
```suggestion
        {% if page.canonical is defined and page.canonical %}
```

I did some tests and this change was enough to make the canonical appear when is was provided in the blog post.

Reply:
fixed